app/scripts/controllers/permissions/specifications.js

```diff
@@ -265,6 +266,16 @@ function validateCaveatAccounts(accounts, getInternalAccounts) {
   });
 }
 
+export const UnrestrictedEthSigningMethods = Object.freeze([
+  'eth_sendRawTransaction',
+  'eth_sendTransaction',
+  'eth_sign',
+  'eth_signTypedData',
+  'eth_signTypedData_v1',
+  'eth_signTypedData_v3',
+  'eth_signTypedData_v4',
+]);
+
```

Comment:
Cannot this be moved alongside the other `EthMethods` in `shared/constants/permission.ts`?

Also, what about using an object like:
```ts
export const UnrestrictedEthSigningMethods = Object.freeze({
  eth_sendRawTransaction: 'eth_sendRawTransaction',
  ...
});
```

Reply:
Ignore this one, just keep the array as is (same as `unrestrictedMethods`)